profiler/wdt_follow_ajax.rst

```diff
@@ -0,0 +1,40 @@
+.. index::
+    single: Profiling: WDT Auto-update after AJAX Request
+
+How to make the Web Debug Toolbar Auto-update after AJAX Requests
+================================================================
+
+For single page applications it would be more convenient if the toolbar 
+showed the information for the most recent AJAX request instead of the 
+initial page load.
+
+By setting the `Symfony-Debug-Toolbar-Replace` header to a value of `1` in the
+AJAX request, the toolbar will be automatically reloaded for the request. The
+header can be set on the response object::
+
+    $response->headers->set('Symfony-Debug-Toolbar-Replace', 1);
+
+Only setting the header during development
```

Comment:
For titles we use a special "title case" where the first letter of most, but not all, words is uppercased. So, this title should be:

```
Only Setting the Header During Development
------------------------------------------
```